src/SOFe/AwaitStd/AwaitStd.php

```diff
@@ -80,10 +81,11 @@ public function consumeNextInteract(Player $player, int $priority = EventPriorit
 	 * @param Generator<mixed, mixed, mixed, T> $promise
 	 * @param U $onTimeout
 	 * @return Generator<mixed, mixed, mixed, T|U>
+	 * @throws Throwable
```

Comment:
Why do we need to add this? `safeRace` is not supposed to throw out.

Reply:
I'm not a fan of writing a generic Throwable throw here, since we don't know if $promise throws. This just creates additional error handling requirement on the user when it is not informative at all.

Reply:
Alright, I'll go ahead and remove it then.